tests/integration/setup.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+set -eux
+
+. tests/integration/functions.sh
+
+sudo systemctl restart postgresql || sudo journalctl -xe
+
+git clone --depth 1 https://pagure.io/koji.git
+
+# Create SSL certs
+git clone --depth 1 https://pagure.io/koji-tools.git
+
+pushd koji-tools
+./src/bin/koji-ssl-admin new-ca --common-name "travisci Koji CA"
```

Comment:
Is there any advantage of using SSL auth instead of plaintext password for the tests?

Reply:
Yes, I don't think we should be exercising the password-based authentication in any CI because that is not "production", and we should exercise the SSL authentication paths instead.

I also wanted to exercise the `koji-ssl-admin` tool specifically because I want to improve that tool and make that the officially documented way to set up SSL with Koji.

In fact I think we should drop password-based authentication code in Koji entirely in order to reduce the user choice, complexity of the codebase, and the QE test matrix.